templates/repo/view_list.tmpl

```diff
@@ -5,9 +5,9 @@
 		<div>{{if and .LatestCommit .LatestCommit.Committer}}{{DateUtils.TimeSince .LatestCommit.Committer.When}}{{end}}</div>
 	</div>
 	{{if .HasParentPath}}
-	<div class="repo-file-line">
-		{{svg "octicon-reply"}} <a class="muted" href="{{.BranchLink}}{{if .ParentPath}}{{PathEscapeSegments .ParentPath}}{{end}}">..</a>
-	</div>
+	<a class="repo-file-line parent-link silenced" href="{{.BranchLink}}{{if .ParentPath}}{{PathEscapeSegments .ParentPath}}{{end}}">
```

Comment:
Maybe `tw-block` or `tw-flex (flex-text-block)` should work, then no need to write the single `width: 100%;` in CSS?

Reply:
Yeah in fact, the 100% can be removed completely. It has `repo-file-line` which has `display: flex` which works. Done that.